### PR TITLE
Update instances of inline damage to the newer format

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/iconics/amiri-level-1.json [hunk 1]
```diff
@@ -528,7 +528,7 @@
                     "type": "vitality"
                 },
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain [[/r 1d8[healing]]]{1d8 Hit Points}.</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain @Damage[1d8[healing]]{1d8 Hit Points}.</p>"
```
Comment (simonward) [inline packs/iconics/amiri-level-1.json:531]:
```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain @Damage[1d8[healing]] Hit Points.</p>"
```

File: packs/iconics/kyra-level-1.json [hunk 2]
```diff
@@ -3231,7 +3231,7 @@
                     "type": "vitality"
                 },
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain [[/r 1d8[healing]]]{1d8 Hit Points}.</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain @Damage[1d8[healing]]{1d8 Hit Points}.</p>"
```
Comment (simonward) [inline packs/iconics/kyra-level-1.json:3234]:
```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> (manipulate)</p>\n<hr />\n<p>A <em>healing potion</em> is a vial of a ruby-red liquid that imparts a tingling sensation as the drinker's wounds heal rapidly. When you drink a <em>healing potion</em>, you regain @Damage[1d8[healing]] Hit Points.</p>"
```

File: packs/howl-of-the-wild-bestiary/virtuosic-lyrebird.json [hunk 3]
```diff
@@ -878,7 +878,7 @@
...
+                    "value": "<p>You snap your fingers and point at a target, which begins to smolder. Make a spell attack roll against the target's AC, dealing 2d4 fire damage on a hit. If the target is within your melee reach, you can choose to make a melee spell attack with the flame instead of a ranged spell attack, which increases all the spell's damage dice to d6s.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes double damage and 1d4 persistent fire damage.</p>\n<p><strong>Success</strong> The target takes full damage.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The initial damage increases by 1d4 and the persistent fire damage on a critical hit increases by 1d4.</p>\n<p>@Damage[(@item.level)d4[persistent,fire]]{Leveled Persistent Fire Damage}</p>\n<p>@Damage[(@item.level)d6[persistent,fire]]{Leveled Persistent Fire Damage (Melee)}</p>"
```
Comment (simonward) [inline packs/howl-of-the-wild-bestiary/virtuosic-lyrebird.json:881]: The instances of "Leveled Persistent Fire Damage" should be "Scaling Persistent Fire Damage"

Comment (ammalagonc) [reply packs/howl-of-the-wild-bestiary/virtuosic-lyrebird.json]: Changes made - check when you get a moment.

**Review by simonward — APPROVED** [review]

**Pull request merged**